src/functions/Coverage.ps1

```diff
@@ -107,30 +107,24 @@ function Convert-UnknownValueToInt {
 function Resolve-CoverageInfo {
     param ([psobject] $UnresolvedCoverageInfo)
 
-    $path = $UnresolvedCoverageInfo.Path
+    $paths = $UnresolvedCoverageInfo.Path
 
     $testsPattern = "*$($PesterPreference.Run.TestExtension.Value)"
     $includeTests = $UnresolvedCoverageInfo.IncludeTests
+    $resolvedPaths = @()
 
     try {
-        $resolvedPaths = & $SafeCommands['Resolve-Path'] -Path $path -ErrorAction Stop |
-            & $SafeCommands['Where-Object'] { $includeTests -or $_.Path -notlike $testsPattern }
+        foreach ($path in $paths) {
+            $resolvedPaths += & $SafeCommands['Resolve-Path'] -Path $path -ErrorAction Stop |
```

Comment:
This is slow, you should output from foreach directly into the variable, or use List and Add. There is a factory method on Pester Factory type to create a list.